lm_eval/tasks/naturalqs.py

```diff
@@ -0,0 +1,41 @@
+from . common import HFTask
+import apache_beam

```

Comment:
I think the underlying datasets code requires Apache Beam, but we don't need to import it ourselves.

Reply:
Right on. Removed the line.